tests/selection.spec.ts

```diff
@@ -1944,3 +1944,77 @@ test('should not be misaligned when the editor container has padding or margin',
 
   expect(count).toBe(3);
 });
+
+// #1159
+test('should be removed when the embed block is selected in native selection', async ({
+  page,
+}) => {
+  const room = await enterPlaygroundRoom(page);
+  const { pageId, frameId } = await initEmptyParagraphState(page);
+  await initThreeParagraphs(page);
+
+  const MOCK_IMAGE_ID = '_e2e_test_image_id_';
+  await page.route(
```

Comment:
Somehow we should have a mock server for this. I will try to do this later

Reply:
Yes, the embed tests can be refactored.